package.json

```diff
@@ -130,6 +131,7 @@
     "build": "node --max-old-space-size=3072 ./scripts/build/build.js",
     "build-docs": "node ./scripts/build-docs.js",
     "check-deps": "node ./scripts/check-deps.js",
-    "spellcheck": "npx -p cspell@4.0.31 cspell {bin,scripts,src}/**/*.js {docs,website/blog,changelog_unreleased}/**/*.md"
+    "spellcheck": "npx -p cspell@4.0.31 cspell {bin,scripts,src}/**/*.js {docs,website/blog,changelog_unreleased}/**/*.md",
+    "svgo": "svgo --pretty --indent=2 --precision=1"
```

Comment:
Why should we output pretty content?

Reply:
Because we are Prettier :P

Reply:
I doesn’t make sense to me. Is the author of these svg files wrote it manually? I supposed not. They are image files like `jpg/png`, we should not care about if they are `prettier` IMO.

And if we do care about it, we should support prettier `svg` by ourselves then.

Reply:
We have Prettier for XML now :)

but SVGO does more than what Prettier would do.. it also cleans it up.

Reply:
You still didn't answer why should we prettier these files?

> I doesn’t make sense to me. Is the author of these svg files wrote it manually? I supposed not. They are image files like jpg/png, we should not care about if they are prettier IMO.

I think we should minify them instead.